content/01-getting-started/02-setup-prisma/02-start-from-scratch-sql.mdx

```diff
@@ -276,20 +276,20 @@ Great, you now created three tables in your database
 
 | Column name | Type           | Primary key | Foreign key | Required | Default            |
 | :---------- | :------------- | :---------- | :---------- | :------- | :----------------- |
-| `id`        | `SERIAL`       | **✔️**         | No          | No       | _autoincrementing_ |
-| `createdAt` | `TIMESTAMP`    | No          | No          | **✔️**      | `now()`            |
-| `title`     | `VARCHAR(255)` | No          | No          | **✔️**      | -                  |
+| `id`        | `SERIAL`       | **✔️**       | No          | No       | _autoincrementing_ |
+| `createdAt` | `TIMESTAMP`    | No          | No          | **✔️**    | `now()`            |
+| `title`     | `VARCHAR(255)` | No          | No          | **✔️**    | -                  |
 | `content`   | `TEXT`         | No          | No          | No       | -                  |
-| `published` | `BOOLEAN`      | No          | No          | **✔️**      | `false`            |
-| `author`    | `INTEGER`      | No          | **✔️**         | **✔️**      | `false`            |
+| `published` | `BOOLEAN`      | No          | No          | **✔️**    | `false`            |
+| `authorId`  | `INTEGER`      | No          | **✔️**       | **✔️**    | `false`            |
 
 **Profile**
 
 | Column name | Type      | Primary key | Foreign key | Required | Default            |
 | :---------- | :-------- | :---------- | :---------- | :------- | :----------------- |
-| `id`        | `SERIAL`  | **✔️**         | No          | No       | _autoincrementing_ |
-| `bio`       | `TEXT`    | No          | No          | **✔️**      | -                  |
-| `user`      | `INTEGER` | No          | **✔️**         | **✔️**      | `false`            |
+| `id`        | `SERIAL`  | **✔️**       | No          | No       | _autoincrementing_ |
+| `bio`       | `TEXT`    | No          | No          | **✔️**    | -                  |
+| `userId`    | `INTEGER` | No          | **✔️**       | **✔️**    | `false`            |
```

Comment:
the default value in this row has a wrong leftover value which also needs to be corrected (sorry this was my bad), you can just replace `false` with `-`